htdocs/core/modules/modFournisseur.class.php

```diff
@@ -814,6 +814,7 @@ public function __construct($db)
 			'cd.fk_commande'    => 'PurchaseOrder*',
 			'cd.fk_parent_line' => 'ParentLine',
 			'cd.fk_product'     => 'IdProduct',
+			'cd.ref'     		=> 'Ref',
```

Comment:
Can you use the label SupplierRef instead of Ref to be consistent in the field name with other screens...

Reply:
Yes, I didn't read correctly the file to use the good label...
By checking this, I saw that in the export part of file for the supplier order lines, it is the "RefSupplier" label which is used for the supplier product references instead of "SupplierRef". Would it be useful to modify this label as well ?

![image](https://github.com/Dolibarr/dolibarr/assets/69631660/2ed39ba6-e7a2-43d9-841d-c9fc2a516045)

![image](https://github.com/Dolibarr/dolibarr/assets/69631660/b17af861-eaeb-4bf2-9549-d8f2521ae742)

I updated my modification in this way.